### ISSUE TITLE
[bug] Getting an error using WalletConnectConnector 

### ISSUE THREAD

**Issue by Sam-DeGuzman** [tmm/wagmi#65]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package Version

0.1.3

### Current Behavior

Tried using the WalletConnectConnector from the module import as stated in the docs as usual, getting an error with a message that it's not a constructor.

### Expected Behavior

Expecting to instantiate the WalletConnectConnector

### Steps To Reproduce

Function for instantiating wallet connectors
```
export const walletConnectors = ({ chainId }) => {
	const rpcUrl =
		chains.find((x) => x.id === chainId)?.rpcUrls?.[0] ??
		chain.mainnet.rpcUrls[0];
	return [
		new InjectedConnector({ chains }),
		new WalletConnectConnector({
			options: {
				infuraId: INFURA_ID,
				qrcode: true,
			},
		}),
		new WalletLinkConnector({
			options: {
				appName: "AppName",
				jsonRpcUrl: `${rpcUrl}/${INFURA_ID}`,
			},
		}),
	];
};
```

### Anything else?

![image](https://user-images.githubusercontent.com/74597218/148773714-1a3f6340-252d-4b51-85fa-292967e3b5d4.png)

**Comment by tmm** [comment]
Were you using a previous version of wagmi? Have you tried uninstalling wagmi and installing again?

**Comment by Sam-DeGuzman** [comment]
Yes, I've tried uninstalling it using `yarn remove wagmi` and reinstall it likewise using `yarn add wagmi`, thanks

**Comment by tmm** [comment]
released `wagmi@0.1.4`

if this is still an issue, feel free to re-open.
